hdbt.theme

```diff
@@ -717,71 +596,12 @@ function hdbt_preprocess_breadcrumb(&$variables): void {
 }
 
 /**
- * Implements template_preprocess_paragraph().
- *
- * @param array $variables
- *   An associative array containing:
- *   - elements: An array of elements to display in view mode.
- *   - paragraph: The paragraph object.
- *   - view_mode: View mode; e.g., 'full', 'teaser'...
+ * Implements hook_preprocess_paragraph__HOOK().
  */
-function hdbt_preprocess_paragraph(array &$variables): void {
-  /** @var \Drupal\paragraphs\Entity\Paragraph $paragraph */
+function hdbt_preprocess_paragraph__image(&$variables) {
   $paragraph = $variables['paragraph'];
   $paragraph_type = $paragraph->getType();
 
-  // Set theme path variable for paragraphs as well.
-  $variables['theme_path'] = _hdbt_get_theme_path();
```

Comment:
This function still gets called on lines 69 and 180 of `hdbt.theme` resulting in errors